doc/operations/troubleshooting.rst

```diff
@@ -0,0 +1,81 @@
+Troubleshooting CCF
+===================
+
+This page contains troubleshooting tips for CCF.
+
+Tips for interacting with CCF to diagnose issues
+------------------------------------------------
+.. note:: In the examples below this documentation uses ``example-ccf-domain.com`` as an example CCF domain, you will need to replace that with your own CCF domain when using these commands. 
+
+.. note:: CCF may be deployed with a loadbalancer which may cache the node which last responded to a query from an IP address. Until the cache clears, any subsequent queries from that IP address will be handled by the same node. As an example, if the cache clears after one minute, then in order to get a response from a different node, an operator must wait one minute between queries.  
```

Comment:
This is correct, but:

1. it's independent from CCF, and completely dependent on whether and how a load balancer is used
3. "will be handled by the same node" is superficially correct, but can easily be interpreted too strongly: nodes can and will forward queries to another nodes depending on endpoint forwarding policies and session state. So while a limited amount of handling will always happen on that same node, the actual execution may not.

Reply:
1. Agreed and hopefully that is clear in the wording.
2. I've clarified the wording to avoid misinterpretation. 
On 1) I think it is useful to have this note as users without a loadbalancer can easily ignore it and those with a loadbalancer may benefit. I can be convinced to drop this note altogether, but otherwise I'll leave it as it is now.